src/codeu/chat/client/simplegui/originalgui/ChatSimpleGui.java

```diff
@@ -0,0 +1,122 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codeu.chat.client.simplegui.originalgui;
```

Comment:
Official google style says 2 spaces (because Java is so wordy).  Official android style says 4 spaces.  Your choice really what you want, but just keep it consistent :)

that said... cathy-style says 2 spaces preferred unless otherwise specified by higher beings ;) of power